src/go/k8s/tests/e2e/admin-api-tls-ext/create-cert-secrets.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CLUSTER_NAME=$1
+KEY_FILE=/tmp/client.key.$$
```

Comment:
nit: Prefer to use something like `$(mktemp -t client.key)` to prevent any potential concurrency or stale file issues from occurring.